utils/ort/src/main/kotlin/storage/S3FileStorage.kt

```diff
@@ -134,7 +132,7 @@ class S3FileStorage(
         runCatching {
             s3Client.putObject(request, body)
         }.onFailure { exception ->
-            if (exception is S3Exception) logger.warn { "Can not write '$path' to S3 bucket '$bucketName'." }
+            if (exception is S3Exception) logger.warn { "Can not write '$path' to S3 bucket '$bucketName'. Error: ${exception.message}" }
```

Comment:
While I acknowledge that the same style is used in original line 99 already, I believe we should change both to the following style:

```kotlin
            if (exception is S3Exception) {
                logger.warn { "Can not write '$path' to S3 bucket '$bucketName': ${exception.collectMessages()}" }
            }
```